meeting-notes/2018/2018-08-13--golang-core-dev-team-weekly.md

```diff
@@ -0,0 +1,184 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 August 13, 2018
+
+- **Lead:** David Dias
+- **Notetaker:** @keks
+- **Attendees:**
+  - @stebalien
+  - @diasdavid
+  - @bigs
+  - @Magik6k
+  - @keks
+  - @djdv
+  - @hsanjuan
+  - @johnnycrunch
+  
+- **Recording:** Recording is currently private to the Go Core Dev Team.
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+
+## Notes
+
+### Week Update
+
+@stebalien
+ - Done:
+   - Reviewed a bunch of PRs
+   - Responded to/triaged a bunch of issues
+   - Fixed a few bugs
+   - Notable: update gogo protobuf
+ - Blocked:
+   - https://github.com/multiformats/multiaddr/pull/68 <- @daviddias P3
+   - https://github.com/multiformats/multiaddr/pull/68#issuecomment-401971665
+ - Next:
+   - More of the same
+     - CIDv1
+   - Finish the protocol negotiation spec
+
+
+@diasdavid
+ - Done:
+   - Kick off the Go Core Dev Weekly Sync
+   - Created https://github.com/ipfs/pm/blob/master/GOLANG_CORE_DEV_MGMT.md
+   - Updated the go-ipfs and go-libp2p waffle boards and created the go-ipld one 
+   - Go Core Dev Team Interviews (ongoing)
+   - Proposed Lead Maintainer Protocol up for discussion
+ - Blocked:
+   - none
+ - Next:
+   - Continue doing the Go Core Dev Team Interviews
+   - Encourage everyone to do some Waffle Grooming
+     - rough time scale: 30min/wk
+
+
+@marten-seemann (not attending)
+ - Done: 
+   - various QUIC improvements
+ - Blocked:
+   - not blocked by anything
+ - Next:
+   - re-evaluate TLS1.3 libraries for their use in QUIC
+   
+@kevina (not attending)
+  - Done:
+    - Add ability to retrieve blocks even if given using a different CID version (https://github.com/ipfs/go-ipfs/pull/5285)
+    - Builder interface for CID https://github.com/ipfs/go-cid/pull/53 & https://github.com/ipfs/go-ipfs/pull/5375
+    - Learned how to use gx-workspace and work with gx dependency managment in general
+    - Various work towards Base32 CidV1 (see Other notes section)
+  - Blocked:
+     - Various aspects of Base32 CidV1 need feedback (see Other notes section)
+  - Next:
+     - Continue Base32 CidV1 work
+     - Maybe add support for inlinling via the id-hash (https://github.com/ipfs/go-ipfs/pull/5281).
+    
+@lgierth (not attending)
+- done
+  - infra stuff, nothing go related
+- blocked
+  - nothing go related :)
+- next
```

Comment:
@lgierth will you have bandwidth soon to pick https://github.com/ipfs/go-ipfs/pull/4595 back up?

Reply:
Yes definitely until end-of-August -- how badly are you blocked?

Reply:
I'm getting ready to do end to end validation, which needs the branch. I started implementing the endpoints in js, but it doesn't currently have refs support. I can look at getting the existing go branch running locally to at least get e2e validation happening. If we can get it live by mid September I think we can still get js peer/content delegated routing out by end of the quarter.